src/engine/plugins/isochrone.cpp

```diff
@@ -0,0 +1,330 @@
+#include "engine/plugins/isochrone.hpp"
+#include "engine/edge_unpacker.hpp"
+#include "engine/plugins/plugin_base.hpp"
+#include "util/timing_util.hpp"
+#include "util/coordinate_calculation.hpp"
+#include "util/log.hpp"
+
+#include "extractor/edge_based_node.hpp"
+
+#include <queue>
+#include <iomanip>
+#include <cmath>
+#include <unordered_set>
+#include <numeric>
+
+namespace osrm
+{
+namespace engine
+{
+namespace plugins
+{
+
+IsochronePlugin::IsochronePlugin() {}
+
+Status
+IsochronePlugin::HandleRequest(const std::shared_ptr<const datafacade::BaseDataFacade> facade,
+                               const api::IsochroneParameters &parameters,
+                               std::string &pbf_buffer) const
+{
+
+    // const auto range = parameters.range;
+    const auto range = 300;
```

Comment:
Is `range` intentionally `signed int`?

Compare with [unsigned IsochroneParameters::range](https://github.com/Project-OSRM/osrm-backend/pull/3652/files#diff-284c45c35e4acbf777ab511606d198e2R53) declaration.

The type difference actually does affect the isochrones result (see my next comment).